### ISSUE TITLE
Clarifying questions broken?

### ISSUE THREAD

**Issue by jalakoo** [AntonOsika/gpt-engineer#802]
## Expected Behavior

Expecting gpt-enginner to ask clarifying questions before building. 

## Current Behavior

`gpt-engineer <folder_path>` currently will attempt to build an app from a prompt file. However, no questions are ever asked.  When using the `--steps clarify` option I get both an `llm_cache` unsupported error and a ValueError detailed below.

## Failure Information
```
/Library/Frameworks/Python.framework/Versions/3.11/lib/python3.11/site-packages/langchain/__init__.py:34: UserWarning: Importing llm_cache from langchain root module is no longer supported. Please use langchain.globals.set_llm_cache() / langchain.globals.get_llm_cache() instead.
  warnings.warn(
Nothing to clarify.
Traceback (most recent call last):

  File "/Library/Frameworks/Python.framework/Versions/3.11/bin/gpt-engineer", line 8, in <module>
    sys.exit(app())
             ^^^^^

  File "/Users/jasonkoo/jason/repos/gpt-engineer/gpt_engineer/cli/main.py", line 169, in main
    messages = step(ai, dbs)
               ^^^^^^^^^^^^^

  File "/Users/jasonkoo/jason/repos/gpt-engineer/gpt_engineer/core/steps.py", line 270, in gen_clarified_code
    messages = AI.deserialize_messages(dbs.logs[clarify.__name__])
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^

  File "/Users/jasonkoo/jason/repos/gpt-engineer/gpt_engineer/core/ai.py", line 349, in deserialize_messages
    return list(messages_from_dict(prevalidated_data))  # type: ignore
                ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^

  File "/Library/Frameworks/Python.framework/Versions/3.11/lib/python3.11/site-packages/langchain/schema/messages.py", line 348, in messages_from_dict
    return [_message_from_dict(m) for m in messages]
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^

  File "/Library/Frameworks/Python.framework/Versions/3.11/lib/python3.11/site-packages/langchain/schema/messages.py", line 348, in <listcomp>
    return [_message_from_dict(m) for m in messages]
            ^^^^^^^^^^^^^^^^^^^^^

  File "/Library/Frameworks/Python.framework/Versions/3.11/lib/python3.11/site-packages/langchain/schema/messages.py", line 336, in _message_from_dict
    raise ValueError(f"Got unexpected message type: {_type}")

ValueError: Got unexpected message type: AIMessageChunk
```

### Steps to Reproduce

If possible, provide detailed steps for reproducing the issue.

1. Ran `gpt-engineer <folder_path_with_prompt_file> --steps clarify

### Failure Logs

Output listed in Failure Information above.

**Comment by ATheorell** [comment]
Thanks for the bug report, we will look into this

**Comment by ATheorell** [comment]
This is now addressed in PR #833  by restricting the langchain version. Reinstalling from main branch source should solve it.
